src/main/sensors/gyro.c

```diff
@@ -657,6 +686,9 @@ static void gyroUpdateSensor(gyroSensor_t *gyroSensor, timeUs_t currentTimeUs)
         for (int axis = 0; axis < XYZ_AXIS_COUNT; axis++) {
             // NOTE: this branch optimized for when there is no gyro debugging, ensure it is kept in step with non-optimized branch
             float gyroADCf = (float)gyroSensor->gyroDev.gyroADC[axis] * gyroSensor->gyroDev.scale;
+            if (axis==FD_YAW && gyroConfig()->overflowTriggerThreshold > 0) {
```

Comment:
I don't like copy/pasting repetitive code blocks into this. I can see the usefulness of the debug / no debug code paths, but I think functions should be extracted as much as possible out of this in order to reduce the chance of divergence. The compiler will take care of inlining them anyway.